src/utils.ts

```diff
@@ -1,13 +1,28 @@
 import type { MessageAction } from './types';
 
+type IframeConnectedMessage = {
```

Comment:
Do you think it would be nice to merge these types with the `MessageAction` enum? It might be annoying maintaining both. Also in future it could be nice to import this type in the web app